base/sysinfo.jl

```diff
@@ -201,6 +227,19 @@ function _cpu_summary(io::IO, cpu::AbstractVector{CPUinfo}, i, j)
     println(io)
 end
 
+"""
+    Sys.cpu_summary(io::IO=stdout, cpu::AbstractVector{CPUinfo}=cpu_info())
+
+Print a summary of CPU information to the `io` stream (defaulting to [`stdout`](@ref)), organizing and displaying aggregated data for CPUs with the same model, for a given array of `CPUinfo` data structures
+describing a set of CPUs (which defaults to the return value of the [`Sys.cpu_info`](@ref) function).
+
+The summary includes aggregated information for each distinct CPU model,
+providing details such as average CPU speed and total time spent in different modes (user, nice, sys, idle, irq) across all cores with the same model.
+
+!!! compat "Julia 1.11"
+    As of Julia 1.11, the `Sys.cpu_summary(io, cpu)` output is replicated by `show(io, "text/plain", cpu)` (which is called by e.g.
+`display` for REPL output), so it need not be called directly.
```

Comment:
```suggestion
     `display` for REPL output), so it need not be called directly.
```
Need to indent the second line too.

(It is nicer if you click the "Commit suggestion" button than if you do the patch manually, because then github should automatically resolve the comment.)